ahk/window.py

```diff
@@ -514,3 +518,51 @@ def find_windows_by_class(self, class_name, exact=False):
     def find_window_by_class(self, *args, **kwargs):
         with suppress(StopIteration):
             return next(self.find_windows_by_class(*args, **kwargs))
+
+    def run_window(self, target: str, working_dir="", options="") -> Window:
+        """Run command
+
+        Arguments:
+            target {str} -- Command
+
+        Keyword Arguments:
+            working_dir {str} -- Working dir (default: {""})
+            options {str} -- Options (default: {""})
+
+        Returns:
+            Window -- Window object
+        """
+        script = self.render_template(
+            "window/run.ahk",
+            command="Run",
+            target=target,
+            working_dir=working_dir,
+            options=options,
+            wait=True
+        )
+        ahk_id = self.run_script(script)
+        return self.find_window_by_id(ahk_id)
+
+    def run(self, target, working_dir="", options=""):
```

Comment:
Would it be possible for this to accept a `wait` keyword argument that would use `RunWait`? (should probably rename the internal name `wait`)?

Alternatively, maybe a `run_wait` function for RunWait would work, too.